test/unit/APITest.php

```diff
@@ -99,21 +108,27 @@ private function getExpectedData($store, $headers, $converted_body, $extra = arr
     public function testTranslationURL()
     {
         list($store, $headers) = StoreAndHeadersFactory::fromFixture('japanese_path_request');
-        $body = '<html></html>';
+        $original_html = '<html></html>';
+        $mock = $this->mockTranslationApi(null);
         $request_options = new RequestOptions(array(), false);
-        $expected_api_url = $this->getExpectedApiUrl($store, $headers, $body, $request_options);
+        $result = API::translate($store, $headers, $original_html, $request_options);
 
-        $this->assertTrue(API::url($store, $headers, $body, $request_options) === $expected_api_url);
+        list($method, $url, $data, $timeout) = $mock->arguments[0];
+        $converted_html = $this->convertHtmlToSendTranslationAPI($original_html, $store, $headers);
```

Comment:
I think you can use `getExpectedHtmlHeadContent` to get the HTML you need?

Reply:
Yes. I fixed it.

Reply:
For example, I thought you would be doing this
```
        $expected_head_content = $this->getExpectedHtmlHeadContent($store, $headers);
        $expected_html_before_send = "<html lang=\"en\"><head>$expected_head_content</head><body><h1>en</h1></body></html>";
```

This is basically equivalent to `this->convertHtmlToSendTranslationApi`.